packages/phalanx/src/Phalanx_any.hpp

```diff
@@ -1,3 +1,12 @@
+// @HEADER
+// *****************************************************************************
+//        Phalanx: A Partial Differential Equation Field Evaluation 
+//       Kernel for Flexible Management of Complex Dependency Chains
+//
+// Copyright 2008 NTESS and the Phalanx contributors.
+// SPDX-License-Identifier: BSD-3-Clause
+// *****************************************************************************
+// @HEADER
```

Comment:
@ccober6 - please do not change the copyright on the Phalanx_any.hpp file. It is copied from boost and must retain the original copyright. This file will be removed soon since we have std::any in c++17.

Reply:
Gotcha.  Restored to original state.